data/Engeniusam.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Samuel Macharia",
+  "type": "personal",
+  "bio": "I am a Frontend Software Engineer.",
+  "avatar": "https://twitter.com/engeniusam/photo",
```

Comment:
Hi @Engeniusam, thank you for contributing  :tada: 
Please delete this line, it is not required anymore :+1: 
```Suggestion
```